db/migrate/20171002234843_create_taxon_child_parents.rb

```diff
@@ -0,0 +1,12 @@
+class CreateTaxonChildParents < ActiveRecord::Migration[5.1]
+  def change
+    create_table :taxon_child_parents do |t|
+      t.integer :taxid
+      t.integer :parent_taxid
+      t.string :rank
+      t.references :taxon_description, foreign_key: true
```

Comment:
i thought we discussed yesterday we don't want to add the version for now. because every time we run this script, a lot of new records would be generated and overtime, it would slow down the database. If we want to keep version, we should probably have another table which archived the old version and move the records over. The actualy TaxonChildParents datababase will only keep the most recent K versions.

Reply:
Also, it's probably a lot more efficient to just do (taxid => lineage) table instead of the child/parent as it has a lot more records.

Reply:
ok, I removed TaxonDescription. Will do full lineage table in a later PR